Runtime/Scripts/Options/CallbackOptions.cs

```diff
@@ -5,11 +5,19 @@
 [Serializable]
 public class CallbackOptions
 {
-    [Tooltip("This callback is invoked when the fracturing/slicing process has been completed.")]
+    [Tooltip("This callback is invoked when the async fracturing/slicing process has been completed.")]
     public UnityEvent onCompleted;
 
+    public delegate void OnFractureHandler(Collider instigator, GameObject fracturedObject, Vector3 point);
+    public event OnFractureHandler onFracture;
+
     public CallbackOptions()
     {
         this.onCompleted = null;
     }
+
+    public void CallOnFracture(Collider instigator, GameObject fracturedObject, Vector3 point)
+    {
+        onFracture(instigator, fracturedObject, point);
```

Comment:
What happens if `onFracture` is null? Does this need a null check?

Reply:
Yep, that is a good point.  The latest has that change.